swagger/swagger.go

```diff
@@ -32,7 +32,6 @@ type Swagger struct {
 	Paths               map[string]*Item    `json:"paths" yaml:"paths"`
 	Definitions         map[string]Schema   `json:"definitions,omitempty" yaml:"definitions,omitempty"`
 	SecurityDefinitions map[string]Security `json:"securityDefinitions,omitempty" yaml:"securityDefinitions,omitempty"`
-	Security            map[string][]string `json:"security,omitempty" yaml:"security,omitempty"`
```

Comment:
From the spec. Security also useful. http://swagger.io/specification/#fixed-fields-15

```
security	[Security Requirement Object]	A declaration of which security schemes are applied for the API as a whole. The list of values describes alternative security schemes that can be used (that is, there is a logical OR between the security requirements). Individual operations can override this definition
```